src/magellan_core/launch/robot_localization.launch

```diff
@@ -0,0 +1,39 @@
+<launch>
+
+    <node pkg="robot_localization" type="ekf_localization_node" name="ekf_localization" clear_params="true">
+      <param name="frequency" value="10"/>  
+      <param name="sensor_timeout" value="0.2"/>  
+      <param name="two_d_mode" value="true"/>
+
+      <param name="map_frame" value="map"/>
+      <param name="odom_frame" value="odom"/>
+      <param name="base_link_frame" value="base_link"/>
+      <param name="world_frame" value="odom"/>
+
+      <param name="odom0" value="odom"/>
+      <param name="imu0" value="/imu/data"/> 
+
+      <rosparam param="odom0_config">[true,  true,  false, 
+                                      false, false, true, 
+                                      true, true, false, 
+                                      false, false, true,
+                                      false, false, false]</rosparam>
+
+      <rosparam param="imu0_config">[false, false, false, 
+                                     true,  true,  true, 
+                                     false, false, false, 
+                                     true,  true,  true,
+                                     true, true, true]</rosparam>
+
+      <param name="odom0_differential" value="true"/>
+      <param name="imu0_differential" value="true"/>
+
+      <param name="imu0_remove_gravitational_acceleration" value="false"/>
+
+      <rosparam param="process_noise_covariance">[LARGE ARRAY with small values]</rosparam>
+
+      <rosparam param="initial_estimate_covariance">[LARGE ARRAY with 1e-9]</rosparam>
```

Comment:
This seems like example text - is this file loaded anywhere?

Reply:
Sorry this is file is not needed anymore. I made when I was first trying to figure out robot_localization